urllib3/exceptions.py

```diff
@@ -196,6 +196,14 @@ class ResponseNotChunked(ProtocolError, ValueError):
     pass
 
 
+class BodyNotHttplibCompatible(AssertionError, ValueError):
```

Comment:
Okay, the new commit stops exceptions in urllib3.exceptions from participating in Python's built-in class hierarchy and adds a new base for urllib3 as @sigmavirus24 suggested.